src/main/java/echoquery/sql/QueryBuilder.java

```diff
@@ -84,17 +92,11 @@ public static QueryBuilder of(Intent intent) {
     } else {
       aggregationFunc = QueryUtil.functionCall(
           SlotUtil.getAggregateFunction(aggregate),
-          QueryUtil.nameReference(aggregationColumn));
+          new QualifiedNameReference(
+              QualifiedName.of(from.wherePrefix(1), aggregationColumn)));
```

Comment:
I'm so confused haha, isn't the prefix of the column just the table the column belongs to?